### PR TITLE
Fix #15

### LINKED ISSUE #15: Mention GUI to import data for area in docs
Importing data is covered here: https://a-b-street.github.io/docs/tech/map/importing/index.html

Easiest way to get new map data is from the GUI, worth mentioning that.

### PR DESCRIPTION
**Pull request by Robinlovelace**
Motivated by https://github.com/a-b-street/abstr/pull/66/commits/cb8d1f6c9ce6fefe321fe91abac97a994582120e

### CODE REVIEW

File: book/src/tech/map/importing/README.md [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+map data is by using the GUI, e.g. as follows:
+
+```bash
+cargo run --bin game --release # launch the game
+```
+
+Once the game has been launched, click on the Sandbox option, and then
+on the button in the top right that allows you to change map:
+
+![](https://user-images.githubusercontent.com/1825120/131584198-c58bccf0-5712-4cd6-bd62-16ba607dc4af.png)
```
Comment (dabreegster) [inline book/src/tech/map/importing/README.md:15]: This is a nifty hosting trick, but it means the contents of this repository no longer include all of the content in the docs. I'll commit this PR, then add a followup commit to explicitly put these images in the repo too.

**Comment by dabreegster** [comment]
Thanks for finding and fixing this!

File: book/src/tech/map/importing/README.md (hunk 1 above)
Comment (Robinlovelace) [reply book/src/tech/map/importing/README.md:15]: > This is a nifty hosting trick, but it means the contents of this repository no longer include all of the content in the docs. I'll commit this PR, then add a followup commit to explicitly put these images in the repo too.

Entirely reasonable. I do it to minimise repo sizes but no benefit in this repo that's meant to host the docs and wants to be self standing so big :+1:

Comment (Robinlovelace) [reply book/src/tech/map/importing/README.md:15]: > I do it to minimise repo sizes

And habit, a quick way to get images up there but 100% reliant on GitHub which may not be the most future proof solution to hosting images (although is pretty good).

**Comment by dabreegster** [comment]
https://a-b-street.github.io/docs/user/new_city.html
Actually I just realized, I think this other page is a better fit. I'll move it over now...

**Comment by Robinlovelace** [comment]
>  Actually I just realized, I think this other page is a better fit. I'll move it over now...

I suggest adding a hyperlink to that `new_city.html` page in the importing page. Lots of cross references will help strengthen the docs.

File: book/src/tech/map/importing/README.md (hunk 1 above)
Comment (dabreegster) [reply book/src/tech/map/importing/README.md:15]: Avoiding file size creep is definitely important -- that's one reason I split this repo out from the code. But I don't want to one day lose the images by accident; being self-contained is safer

**Pull request merged**